lib/Test2/Harness/Renderer/JUnit.pm

```diff
@@ -87,10 +103,12 @@ sub render_event {
     # We have all the data. Print the XML.
     if ( $f->{'harness_job_end'} ) {
         $self->close_open_failure_testcase( $test, -1 );
-        $test->{'stop'} = $event->{'stamp'};
+        $test->{'stop'} = $stamp;
         $test->{'testsuite'}->{'time'} = $test->{'stop'} - $test->{'start'};
+        $test->{'testsuite'}->{'timestamp'} = _timestamp( $test->{'start'} );
```

Comment:
Your last commit needs to be optional. it confuses the heck out of jenkins if I recall correctly which is why I took it out.

Reply:
Do you mean the timestamp needs to be optional or the classname? You commented on this line but the last commit was to add the classname.

Reply:
@toddr ping

Reply:
ping @toddr